roles/matrix-bot-maubot/tasks/setup_install.yml

```
@@ -0,0 +1,113 @@
+---
+
+- name: Ensure maubot paths exist
+  file:
+    path: "{{ item.path }}"
+    state: directory
+    mode: 0755
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_groupname }}"
+  with_items:
+    - {path: "{{ matrix_bot_maubot_base_path }}", when: true}
+    - {path: "{{ matrix_bot_maubot_data_path }}", when: true}
+    - {path: "{{ matrix_bot_maubot_docker_src_files_path }}", when: true}
```

`matrix_bot_maubot_docker_src_files_path` only needs to be created when `matrix_bot_maubot_container_image_self_build` is `true`.

I imagine you need 

```suggestion
    - {path: "{{ matrix_bot_maubot_docker_src_files_path }}", when: "{{ matrix_bot_maubot_container_image_self_build }}"}
```